src/test/test-web-util.c

```diff
@@ -10,7 +10,7 @@
 
 static void test_is_valid_documentation_url(void) {
         assert_se(documentation_url_is_valid("http://www.freedesktop.org/wiki/Software/systemd"));
-        assert_se(documentation_url_is_valid("https://www.kernel.org/doc/Documentation/binfmt_misc.txt"));
+        assert_se(documentation_url_is_valid("https://www.kernel.org/doc/Documentation/admin-guide/binfmt-misc.rst"));
```

Comment:
in the systemd-binfmt.service.in's Documentation setting we use the URL `https://www.kernel.org/doc/html/latest/admin-guide/binfmt-misc.html` (i.e. html rather than rst). In the man page we link to the rst page otoh. i guess we have some house cleaning to do ...